lib/src/Uncertainty/Algorithm/OrthogonalBasis/OrthogonalProductPolynomialFactory.cxx

```diff
@@ -304,4 +304,16 @@ Sample OrthogonalProductPolynomialFactory::getNodesAndWeights(const Indices & de
   return nodes;
 }
 
+/* Get marginal functions */
```

Comment:
It should be made clear that this `getMarginal()` method returns the family of orthogonal polynomials restricted to a given list of _input_ variables.

Reply:
Fixed at 27dd0a2f5db6d256c1e1654bafb28a8976488d00